route/dashboard.js

```diff
@@ -0,0 +1,176 @@
+'use strict'
+
+const Router = require('express').Router
+const marketData = require('../lib/market-data')
+const db = require('../lib/stats-db')
+const converterStats = require('./stats/converter-stats')
+const auctionsStats = require('./stats/auction-stats')
+const moment = require('moment-timezone')
+const router = new Router()
+
+function parseFromTo(req, res, next) {
+    moment.tz.setDefault('UTC')
+    const { query } = req
+    const birthTime = 1529280000 // in seconds
```

Comment:
Perhaps is better to have this in the config.

Reply:
Good idea.